framework/wazuh/core/engine/__init__.py

```diff
@@ -0,0 +1,56 @@
+from contextlib import asynccontextmanager
+from logging import getLogger
+from typing import AsyncIterator
+
+from httpx import AsyncClient, AsyncHTTPTransport, ConnectError, Timeout, TimeoutException, UnsupportedProtocol
+from wazuh.core.exception import WazuhEngineError
+
+logger = getLogger('wazuh')
+
+# TODO: use actual default path
```

Comment:
I'm making a list with these TODOs so we can create a technical debt issue to solve them later